<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowMinimal</class>
 <widget class="QMainWindow" name="MainWindowMinimal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="message"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QListWidget" name="followedChannelsList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="usernameLabel">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="usernameLineEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="twitchBtn">
        <property name="text">
         <string>Twitch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="switchBtn">
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateBtn">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowMinimal</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
